USE_CASES.md

```diff
@@ -174,6 +176,217 @@ print(response.body)
 print(response.headers)
 ```
 
+<a name="create-a-django-app-to-send-email-with-sendgrid"></a>
+# Create a Django app to send email with SendGrid
+
+This tutorial explains how we set up a simple Django app to send an email with the SendGrid Python SDK and how we deploy our app to Heroku.
+
+## Create a Django project
+
+We first create a project folder.
+
+```bash
+$ mkdir hello-sendgrid
+$ cd hello-sendgrid
+```
+
+We assume you have created and activated a [virtual environment](https://virtualenv.pypa.io/) (See [venv](https://docs.python.org/3/tutorial/venv.html) for Python 3+) for isolated Python environments.
+
+Run the command below to install Django, Gunicorn (a Python WSGI HTTP server), and SendGrid Python SDK.
+
+```bash
+$ pip install django gunicorn sendgrid
+```
+
+It's a good practice for Python dependency management. We'll pin the requirements with a file `requirements.txt`.
+
+```bash
+$ pip freeze > requirements.txt
+```
+
+Run the command below to initialize a Django project.
+
+```bash
+$ django-admin startproject hello_sendgrid
+```
+
+The folder structure should look like this:
+
+```
+hello-sendgrid
+├── hello_sendgrid
+│   ├── hello_sendgrid
+│   │   ├── __init__.py
+│   │   ├── settings.py
+│   │   ├── urls.py
+│   │   └── wsgi.py
+│   └── manage.py
+└── requirements.txt
+```
+
+Let's create a page to generate and send an email to a user when you hit the page.
+
+We first create a file `views.py` and put it under the folder `hello_sendgrid/hello_sendgrid`. Add the minimum needed code below.
+
+```python
+import os
+
+from django.http import HttpResponse
+
+import sendgrid
+from sendgrid.helpers.mail import *
+
+
+def index(request):
+    sg = sendgrid.SendGridAPIClient(
+        apikey=os.environ.get('SENDGRID_API_KEY')
+    )
+    from_email = Email('test@example.com')
+    to_email = Email('test@example.com')
+    subject = 'Sending with SendGrid is Fun'
+    content = Content(
+        'text/plain',
+        'and easy to do anywhere, even with Python'
+    )
+    mail = Mail(from_email, subject, to_email, content)
+    response = sg.client.mail.send.post(request_body=mail.get())
+
+    return HttpResponse('Email Sent!')
+```
+
+**Note:** It would be best to change your to email from `test@example.com` to your own email, so that you can see the email you receive.
+
+Now the folder structure should look like this:
+
+```
+hello-sendgrid
+├── hello_sendgrid
+│   ├── hello_sendgrid
+│   │   ├── __init__.py
+│   │   ├── settings.py
+│   │   ├── urls.py
+│   │   ├── views.py
+│   │   └── wsgi.py
+│   └── manage.py
+└── requirements.txt
+```
+
+Next we open the file `urls.py` in order to add the view we have just created to the Django URL dispatcher.
+
+```python
+from django.conf.urls import url
+from django.contrib import admin
+
+from .views import index
+
+
+urlpatterns = [
+    url(r'^admin/', admin.site.urls),
+    url(r'^$', index, name='sendgrid'),
```

Comment:
With this code, I get an error when trying to go to http://127.0.0.1:8000/sendgrid/: https://screencast.com/t/NrcQ2pHAtWi

But if I use `url(r'^sendgrid/', index, name='sendgrid')` it works fine.

Reply:
@thinkingserious Thank you for the review! I've revised the text and removed the part I think it may confuse readers.